File: backend/src/BABA.Domain/Models/Controle.cs
namespace BABA.Domain.Models
{
    public class Controle
    {
        public int ControleId { get; set; }
        public string Nome { get; set; }
        public bool ehAtleta { get; set; }
        public int CategoriaId { get; set; }
        public int Status { get; set; }
        public Categoria Categoria { get; set; }
        public DateTime DataCadastro { get; set; }
        public int MensalidadeId { get; set; }
        public Mensalidade Mensalidade { get; set; }
    }
}

File: frontend/web/BABA-WEB/src/app/models/controle.ts
import { Categoria } from "./categoria";
import { Mensalidade } from "./mensalidade";

export interface Controle {
  controleId: number;
  nome: string;
  ehAtleta: boolean;
  categoriaId: number;
  status: number;
  categoria: Categoria;
  dataCadastro: Date;
  dataPagamento: Date;
  mensalidadeId: number;
  mensalidade: Mensalidade;
  preco: number;
}
